### ISSUE TITLE
Display avatars of member who replied to a comment

### ISSUE THREAD

**Issue by kdembler** [Joystream/atlas#2799]
## Context

- #1986
⚠️ blocked by 
- https://github.com/Joystream/joystream/issues/4020 

## Designs
Designs include displaying avatar of users who have replied to a given comment (without extending replies themselves), as seen here: https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2555%3A452099 

Because of technical implementations this was left out of scope for the initial MVP, but we should bring this back at some point

## Scope

- [ ] Implement avatars as per designs

**Comment by traumschule** [comment]
looks implemented: https://l1.media/video/6
